src/cmd_all/src/playground.rs

```diff
@@ -26,13 +25,12 @@ use risedev::{
     CompactorService, ComputeNodeService, ConfigExpander, FrontendService, HummockInMemoryStrategy,
     MetaNodeService, ServiceConfig,
 };
+use risingwave_common::config::load_config;
 use tokio::fs::File;
 use tokio::io::AsyncReadExt;
 use tokio::signal;
 
-async fn load_risedev_config(
-    profile: &str,
-) -> Result<(Vec<String>, HashMap<String, ServiceConfig>)> {
+async fn load_risedev_config(profile: &str) -> Result<(Option<String>, Vec<ServiceConfig>)> {
```

Comment:
Previously the first return value is not useful. The new first return value is only for playground.